databricks/koalas/plot.py

```diff
@@ -514,6 +512,27 @@ def _make_plot(self):
         super(KoalasBarhPlot, self)._make_plot()
 
 
+class KoalasKdePlot(KdePlot):
```

Comment:
This seems it has to implement the logic with Koalas or Spark DataFrame operations rather then just simply sampling.